core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -2512,4 +2512,10 @@ private CarbonCommonConstants() {
    * property which defines the presto query default value
    */
   public static final String IS_QUERY_FROM_PRESTO_DEFAULT = "false";
+
+  @CarbonProperty(dynamicConfigurable = true)
```

Comment:
please update in the document also

Reply:
agree with Ajantha, please sync to doc https://github.com/apache/carbondata/blob/master/docs/configuration-parameters.md#dynamic-configuration-in-carbondata-using-set-reset

Reply:
done